### DISCUSSION TITLE
在後台執行時偵測不到遊戲

### DISCUSSION THREAD

**Discussion by spark3515** [moesnow/March7thAssistant#95] [Q&A] [answered]
使用"远程本地多用户桌面"在後台執行時偵測不到遊戲，會重複啟動遊戲
![err](https://github.com/moesnow/March7thAssistant/assets/155278058/f9d2a7b6-229a-4e8e-a782-4ecbcd4d13ed)

**Comment by moesnow** [comment, marked as answer]
游戏语言只支持简体中文
